<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_startServer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить сервер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stopServer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить сервер</string>
    </property>
   </widget>
   <widget class="QWidget" name="drawWidget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="settings">
     <property name="title">
      <string>Настройки</string>
     </property>
     <addaction name="variables"/>
    </widget>
    <addaction name="settings"/>
   </widget>
   <addaction name="menu_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="variables">
   <property name="text">
    <string>Переменные</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
